eng/pipelines/templates/steps/analyze.yml

```diff
@@ -26,6 +26,24 @@ steps:
       python find_invalid_versions.py --always-succeed --service=${{parameters.ServiceDirectory}}
     displayName: Find Invalid Versions
 
+  - pwsh: |
+      Get-ChildItem .\ -Filter "*.py" |
```

Comment:
You should either pass in the base directly instead of relying on the current directory which could be different depending on other steps. Or you can explicitly pass the workingdirectory as an option to this step.